### PR TITLE
change the name of the Envoy's container in egress gateway

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
to istio-proxy

according to https://github.com/istio/istio/pull/7566

### CODE REVIEW

File: content/docs/examples/advanced-egress/egress-gateway/index.md [hunk 1]
```diff
@@ -669,7 +669,7 @@ The output should be the same as in the previous section.
     counter is:
```
Comment (rcaballeromx) [inline content/docs/examples/advanced-egress/egress-gateway/index.md]: Add a reference to the code in line 667:
... of the `istio-proxy` egress gateway proxy ...

Comment (vadimeisenbergibm) [reply content/docs/examples/advanced-egress/egress-gateway/index.md]: Here the meaning is "the proxy of the egress gateway". What happened is just a change in the name istio/istio#7566 - the name of the proxy of the egress gateway changed from `egressgateway` to `istio-proxy`, it is still "the proxy of the egress gateway". The name of "the proxy of the egress gateway" is a minor detail, it is not relevant for the users who will just copy&paste that command to get the statistics of the proxy of the egress gateway.

**Comment by gyliu513** [comment]
@vadimeisenbergibm can you rebase?

**Comment by vadimeisenbergibm** [comment]
@gyliu513 Will do, waiting for 1.0.1 to be released so this PR will become relevant.

File: content/docs/examples/advanced-egress/egress-gateway/index.md (hunk 1 above)
Comment (rcaballeromx) [reply content/docs/examples/advanced-egress/egress-gateway/index.md]: True, but it is relevant to the reader trying to understand the text and the sample code. I agree that it is a minor thing which is why I left it as a comment only. 😄

**Comment by vadimeisenbergibm** [comment]
@gyliu513 I have rebased the PR and verified that it works with 1.0.1. Could you please approve and merge?

**Comment by vadimeisenbergibm** [comment]
@rcaballeromx could you please approve this PR?

**Pull request merged**